<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deletePreset</class>
 <widget class="QWidget" name="deletePreset">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Preset</string>
  </property>
  <widget class="QComboBox" name="combo_preset">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_del">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>deletePreset</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
